test/trento/infrastructure/commanded/event_handlers/sap_system_database_health_event_handler_test.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Trento.Infrastructure.Commanded.EventHandlers.SapSystemDatabaseHealthEventHandlerTest do
+  use ExUnit.Case
+  use Trento.DataCase
+
+  import Mox
+  import Trento.Factory
+
+  alias Trento.Databases.Events.DatabaseHealthChanged
+  alias Trento.Infrastructure.Commanded.EventHandlers.SapSystemDatabaseHealthEventHandler
+  alias Trento.SapSystems.Commands.UpdateDatabaseHealth
+
+  setup [:set_mox_from_context, :verify_on_exit!]
+
+  test "should dispatch DeregisterSapSystem commands when a database is deregistered" do
```

Comment:
The title here seems wrong, missed copy paste replacement?

Reply:
yep, fixed thanks!